ksql-engine/src/main/java/io/confluent/ksql/metrics/MetricCollector.java

```diff
@@ -44,4 +44,15 @@ default void configure(Map<String, ?> map) {  }
   String getId();
 
   Collection<TopicSensors.Stat> stats(String topic);
+
+  /**
+   * Get the current message production across all topics tracked by this collector.
+   */
+  double currentMessageProductionRate();
```

Comment:
you could swap these out for 'defaults'
  default double currentMessageConsumptionRate() {
        return 0;
  }